docs/app/Components/ComponentDoc/ComponentProps/ComponentPropsEnum.js

```diff
@@ -7,7 +7,9 @@ import ComponentPropsExtra from './ComponentPropsExtra'
 import ComponentPropsToggle from './ComponentPropsEnumToggle'
 import ComponentPropsValue from './ComponentPropsEnumValue'
 
-const ComponentPropsEnum = ({ limit, showAll, toggle, values }) => {
+const ComponentPropsEnum = ({ limit, showAll, toggle, type, values }) => {
```

Comment:
The `limit` was added because of the `Icon` names and `Flag` names.  However, those props do not show enums as they are `{custom}`.  I almost want to just show all enums and do away with the state.

At the least, let's increase the limit to `50`.  Most examples have much less than 50 enums, and 50 will still fit OK.  Here are a few examples of our larger lists that show we can easily work with a limit of 50:

**Grid.Row**
`color` is especially funny as it a very common prop and we hide only 3 of them 😂 
![http://g.recordit.co/27LVhsq9Rj.gif](http://g.recordit.co/27LVhsq9Rj.gif)

**Transition**
![http://g.recordit.co/JYMjLC3xIP.gif](http://g.recordit.co/JYMjLC3xIP.gif)

Reply:
Thanks for reminding, `customPropTypes.suggest` should be also parsed as `enum`, I will update it separate PR.